src/email/templating.rs

```diff
@@ -1,19 +1,25 @@
 use crate::http::structs::AybConfigEmail;
-
-const CLI_CONFIRM_TMPL: &str = "To complete your registration, type\n\tayb client confirm {token}";
-const WEB_CONFIRM_TMPL: &str = "To complete your registration, visit\n\t {url}";
+use crate::templating::TemplateString;
 
 pub fn render_confirmation_template(config: &AybConfigEmail, token: &str) -> String {
+    let cli_confirm_tmpl: TemplateString =
```

Comment:
Would it be possible to still keep them module-level CLI_CONFIRM_TMPL so we don't have to compile them on every render?

Reply:
I'm not sure whether I could make the struct const... I'll try, in either case this should be heavily optimized by the compiler

Reply:
Looks like I can't make it const without polluting the code with lifetimes everywhere. This shouldn't have a huge performance overhead anyway :)